src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2341,6 +2350,15 @@ theorem getLsbD_rotateRight {x : BitVec w} {r i : Nat} :
   · simp
   · rw [← rotateRight_mod_eq_rotateRight, getLsbD_rotateRight_of_le (Nat.mod_lt _ (by omega))]
 
+@[simp]
+theorem getElem_rotateRight {x : BitVec w} {r i : Nat} (h : i < w) :
+    (x.rotateRight r)[i] =
+      cond (i < w - (r % w))
+      (x.getLsbD ((r % w) + i))
```

Comment:
This can be a `getElem` as well, right? You might have to change the `cond` to an `if` so that `omega` can see the condition.

Reply:
Done.